client-projects/the-lumery/scss/nav.scss

```diff
@@ -0,0 +1,62 @@
+.nav__dropdown-link {
+  // these will be animated in using 'rotateX'
+  transform-origin: top;
+
+  .background-color-green & {
+    background-color: var(--color-green-light);
+  }
+  .background-color-purple & {
+    background-color: var(--color-purple-light);
+  }
+  .background-color-red & {
+    background-color: var(--color-red-light);
+  }
+
+  &:first-child {
+    padding-top: 1rem;
+  }
+  &:last-child {
+    padding-bottom: 1.2rem;
+  }
+
+  &.w-dropdown-link.w--current {
+    font-weight: 500;
+    pointer-events: none;
+  }
+
+  &:hover {
+    text-decoration: underline;
+    text-decoration-thickness: 2px;
+    text-underline-offset: 0.12em;
+  }
+}
+
+.nav__dropdown-title {
+  .yesterday-page &--yesterday,
+  .today-page &--today,
+  .tomorrow-page &--tomorrow {
+    font-weight: 700;
+  }
+}
```

Comment:
Neat! Though it took me a minute to get my head around it. It outputs as 
```css
.yesterday-page .nav__dropdown-title--yesterday, 
.today-page .nav__dropdown-title--today, 
.tomorrow-page .nav__dropdown-title--tomorrow {
  font-weight: 700;
}
```
right?

Reply:
correct. Admittedly we have the old story of not being able to lookup the code for `.nav__dropdown-title--yesterday` but I wanted to make it clear that the important part was the 'yesterday page' and 'yesterday modifier' connection.